test/Interop/Cxx/extern-var/extern-var-silgen.swift

```diff
@@ -6,7 +6,9 @@ func getCounter() -> CInt {
   return counter
 }
 
+// CHECK: clang name: counter
```

Comment:
```suggestion
// CHECK: // clang name: counter
```

And everywhere else in the PR please. Otherwise the check lines are a bit confusing to read.